components/gallery/GalleryItemAction/GalleryItemActionSection.vue

```diff
@@ -39,7 +39,7 @@ watchEffect(async () => {
   const chainSymbol = urlPrefix.value === 'rmrk' ? unit.value : symbol
 
   priceChain.value = `${parseFloat(price).toFixed(2)} ${chainSymbol}`
-  priceUsd.value = `$${Math.round(Number(price) * Number(ksmPrice))}`
```

Comment:
it's here to show "$" sign

Reply:
doesnt the issue mention to remove the dollar sign since we show usd already?

Reply:
Aight, then we don't need template literals here.
Should we keep "$" sign in `GalleryItemOffers` too?

Reply:
> Should we keep "$" sign in GalleryItemOffers too?

USD